app/controllers/admin/content/system-images.js

```diff
@@ -0,0 +1,36 @@
+import Ember from 'ember';
+import { eventTopics } from 'open-event-frontend/utils/dictionary/event';
+import { keys } from 'lodash';
+
+const { Controller, computed } = Ember;
+
+export default Controller.extend({
+  imageTopic: keys(eventTopics)[0],
+
+  topics: computed(function() {
+    return keys(eventTopics);
+  }),
+
+  imageObjects: computed('imageTopic', function() {
+    let arr = [];
+    let topics = eventTopics[this.get('imageTopic')];
+    let model = this.get('model');
+    model.forEach(function(item) {
```

Comment:
You said that I need to loop model, so that can be done when we return the filtered objects( on basic of topic ) from the route to the template. This can be done when we have the "imageTopic", "dictionary" and the "subtopics objects" at a place. I could have filtered the objects before returning but the thing is I cannot get the imageTopic from the controller into the route.